topi/tests/python/test_topi_bitserial_conv2d_rasp.py

```diff
@@ -45,6 +44,9 @@ def test_bitserial_conv2d():
     verify_bitserial_conv2d_nhwc(1, in_size, ic, oc, k, stride, pad, 1, 1, False)
     verify_bitserial_conv2d_nhwc(1, in_size, ic, oc, k, stride, pad, 2, 1, False)
 
+    verify_bitserial_conv2d_nhwc(1, in_size, ic, oc, k, stride, pad, 1, 1, True)
```

Comment:
Is it possible to add a correctness check here for random or precomputed inputs? It looks like `generate_quantized_np` is no longer used.

Reply:
Added, but it will only run on an arm device.